config/field_test.yml

```diff
@@ -28,10 +28,42 @@
 ################################################################################
 experiments:
   # NOTE: Our feed strategy testing experiment must begin with "feed_strategy"
+  feed_strategy_starting_20221122:
+    # NOTE: Required as we want only want to consider for conversion events that
+    #       occurred on or after the given start_date.
+    started_at: 2022-11-07
```

Comment:
**Question:** IIRC wouldn't this date need to reflect the date this new experiment starts? In other words it should be something like this instead

```suggestion
    started_at: 2022-11-23
```